website/docs/tutorials/08-cairo-1-support/02-project-initialization.md

```diff
@@ -14,100 +16,285 @@ protostar init-cairo1 my-project
 After running the command, the following structure will be generated:
 
 ```
-my-project/
-├── src/
-│   └── main.cairo
-│   └── lib.cairo
+my_project/
+├── hello_starknet/
+│   ├── src/
+│   │   ├── business_logic/
+│   │   │   └── utils.cairo
+│   │   ├── contracts/
+│   │   │   └── hello_starknet.cairo
+│   │   ├── business_logic.cairo
+│   │   ├── contracts.cairo
```

Comment:
Can users have multiple contracts in contracts? If not we shouldn't use plural

Reply:
It it is there to be future proof, we should add an explicit  warning

Reply:
You cannot have multiple contracts anywhere within the crate. I'll add a warning.